src/applications/vaos/appointment-list/components/CommunityCareAppointmentDetailsPage.jsx

```diff
@@ -123,7 +123,7 @@ function CommunityCareAppointmentDetailsPage({
         {!!appointment.comment && (
           <div className="vads-u-flex--1 vads-u-margin-bottom--2 vaos-u-word-break--break-word">
             <h2 className="vads-u-font-size--base vads-u-font-family--sans vads-u-margin-bottom--0">
-              Special instructions
+              You shared these details about your concern
```

Comment:
Actually, I think this text should go back to what it was for the community care details page. The text we show here isn't from the patient as far as I know, it's instructions from the scheduler/provider. @outerpress

Reply:
Good catch. These instructions come from the `instructionsToVeteran` property so it does appear to come from the provider.

We request optional additional comments in the CC flow but I noticed that they aren't displayed in the appointment list details page or the new appointment flow confirmation page. I think the only page those comments are displayed are in the review page. Not sure how important those additional comments are for CC appointments but the new copy could apply to those comments. @outerpress

Reply:
@LeKeve They are shown on the details/confirmation pages, but most of our mocks don't have them set up, since they're stored outside of the main CC request object, in a messages list.

Another thing with VAOS is that requests and the eventual confirmed appointment are separate entities, not one entity that changes status. So something like comments made on a request for a CC or VA appointment won't typically carry over to the created appointment. For confirmed appointments, we only expect patient comments in one situation, where the patient has direct scheduled a VA appointment.

Reply:
Ah, gotcha. 👍